sample/src/main/res/layout/activity_nested_scroll.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why not `LinearLayout`? The test here is about `NestedScrollView` not the parent layout